sources/packages/api/src/database/entities/student-scholastic-standing.model.ts

```diff
@@ -7,7 +7,13 @@ import {
   PrimaryGeneratedColumn,
   RelationId,
 } from "typeorm";
-import { Application, Note, StudentAssessment, User } from ".";
+import {
+  Application,
+  EducationProgramOffering,
+  Note,
+  StudentAssessment,
+  User,
+} from ".";
```

Comment:
Can we please import them from "../../database/entities"

Reply:
what is the advantage of this? bot are relative import

Reply:
No need to change it, please keep it as it is.

Reply:
Yes you are right, but I see we import like both the ways in our codebase, so to be consistent with one used more, asked to change. No need to change.

Reply:
Vue application is using absolute with "@".
API is using relative, **always**.
The confusion here seems to be that "." it is not relative, but it is.